cql3/expr/expression.cc

```diff
@@ -593,15 +594,15 @@ value_list get_IN_values(const ::shared_ptr<term>& t, size_t k, const query_opti
     if (auto dv = dynamic_pointer_cast<lists::delayed_value>(t)) {
         // Case `(a,b) in ((1,1),(2,2),(3,3))`.  Get kth value from each term element.
         const auto result_range = dv->get_elements()
-                | transform(std::bind_front(get_kth, k, options)) | non_null | deref;
-        return to_sorted_vector(move(result_range), comparator);
+                | boost::adaptors::transformed(std::bind_front(get_kth, k, options)) | non_null | deref;
+        return to_sorted_vector(std::move(result_range), comparator);
     } else if (auto mkr = dynamic_pointer_cast<tuples::in_marker>(t)) {
         // Case `(a,b) IN ?`.  Get kth value from each vector<bytes> element.
         const auto val = static_pointer_cast<tuples::in_value>(mkr->bind(options));
         const auto split_values = val->get_split_values(); // Need lvalue from which to make std::view.
         const auto result_range = split_values
-                | transform([k] (const std::vector<bytes_opt>& v) { return v[k]; }) | non_null | deref;
-        return to_sorted_vector(move(result_range), comparator);
+                | transformed([k] (const std::vector<bytes_opt>& v) { return v[k]; }) | non_null | deref;
```

Comment:
In every other case, `transform` was translated to `boost::adaptors::transformed`, but here it's just `transformed` and I don't see why. Is it an oversight, or did I miss a `using boost::adaptors` somewhere? Perhaps the fully qualified name should be used for consistency.

Reply:
I prefer the fully qualified name too, will change.